src/targets/chrome/docker.js

```diff
@@ -40,13 +41,41 @@ const waitOnCDPAvailable = port =>
     );
   });
 
+const getNetworkHost = async dockerId => {
+  let host = '127.0.0.1';
+
+  // https://tuhrig.de/how-to-know-you-are-inside-a-docker-container/
+  const runningInsideDocker = (await fsExtra.exists('/proc/1/cgroup')) &&
```

Comment:
Think we can use the sync version of these functions instead. We won't be blocking anything else anyway.